layout-server/src/main/java/org/finos/vuu/layoutserver/service/LayoutService.java

```diff
@@ -0,0 +1,53 @@
+package org.finos.vuu.layoutserver.service;
+
+import java.util.Date;
+import java.util.List;
+import java.util.UUID;
+import lombok.RequiredArgsConstructor;
+import org.finos.vuu.layoutserver.model.Layout;
+import org.finos.vuu.layoutserver.model.Metadata;
+import org.finos.vuu.layoutserver.repository.LayoutRepository;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@RequiredArgsConstructor
+@Service
+public class LayoutService {
+
+    private final LayoutRepository layoutRepository;
+    private final MetadataService metadataService;
+
+    public Layout getLayout(UUID id) {
+        return layoutRepository.findById(id).orElseThrow();
+    }
+
+    public List<Metadata> getMetadata() {
+        return metadataService.getMetadata();
+    }
```

Comment:
I think it would make more sense to inject the metadata service in the controller, and avoid having this method here.